From ba1480800d1844e1fb5bc628d7df2f45cfd7c715 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?S=C3=A9rgio=20M=2E=20Basto?= <sergio@serjux.com>
Date: Sun, 14 Jan 2024 17:27:57 +0000
Subject: [PATCH 3/4] Fix doc generation

error: An internal exception has occurred.(java.lang.IllegalStateException: "END_ELEMENT) -
---
 core/src/org/apache/commons/lang/Entities.java | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/src/org/apache/commons/lang/Entities.java b/core/src/org/apache/commons/lang/Entities.java
index 20745ab48..d4ef6146c 100644
--- a/core/src/org/apache/commons/lang/Entities.java
+++ b/core/src/org/apache/commons/lang/Entities.java
@@ -23,10 +23,10 @@ import java.util.TreeMap;
  * <p>Provides HTML and XML entity utilities.</p>
  *
  * @see <a href="http://hotwired.lycos.com/webmonkey/reference/special_characters/">ISO Entities</a>
- * @see </br><a href="http://www.w3.org/TR/REC-html32#latin1">HTML 3.2 Character Entities for ISO Latin-1</a>
- * @see </br><a href="http://www.w3.org/TR/REC-html40/sgml/entities.html">HTML 4.0 Character entity references</a>
- * @see </br><a href="http://www.w3.org/TR/html401/charset.html#h-5.3">HTML 4.01 Character References</a>
- * @see </br><a href="http://www.w3.org/TR/html401/charset.html#code-position">HTML 4.01 Code positions</a>
+ * @see <br><a href="http://www.w3.org/TR/REC-html32#latin1">HTML 3.2 Character Entities for ISO Latin-1</a>
+ * @see <br><a href="http://www.w3.org/TR/REC-html40/sgml/entities.html">HTML 4.0 Character entity references</a>
+ * @see <br><a href="http://www.w3.org/TR/html401/charset.html#h-5.3">HTML 4.01 Character References</a>
+ * @see <br><a href="http://www.w3.org/TR/html401/charset.html#code-position">HTML 4.01 Code positions</a>
  *
  * @author <a href="mailto:alex@purpletech.com">Alexander Day Chaffee</a>
  * @author <a href="mailto:ggregory@seagullsw.com">Gary Gregory</a>
-- 
2.43.0

